en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md

```diff
@@ -0,0 +1,131 @@
+# Working with Alerts
+![alert config page]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+
+One of the most useful features in APIM cloud analytics is the **Alerting support**. New analytics solution allows
```

Comment:
One of the most useful features in APIM cloud analytics is **Alerting**. You can configure several types of alerts and receive alerts either within the portal or as emails. These alerts allow you to proactively monitor your API ecosystem and take corrective measures for any abnormalities you find. 
		
You can configure alerts for each environment within your organization separately. Alerts are configured per API and you can also modify/remove already-added alert configurations. 
		
Alerts are subjected to a suppression policy to ensure you are not overwhelmed with duplicate alert notifications. For each alert rule that you configure, there is a 10-minute suppression window applied by default. The suppression interval is currently not configurable. Also, note that suppression works per the alert rule. 

See the details given below about different alert types and how they are configured.

Reply:
Fixed with d555707